content/pages/meetings/2017-11-06_fr.md

```diff
@@ -8,30 +8,40 @@ Slug: meetings/2017-11-06
 Lang: fr
 Status: hidden
 
+<br><br>
```

Comment:
I'm not a big fan of `<br><br>`...

Reply:
I know... But otherwise the whole thing sticks to the top line (which is the language switcher)

Reply:
[I know](https://github.com/geomapfish/geomapfish_website/issues/15#issuecomment-312617702)... Let's do it as it is